ResourceStatusSystem/Utilities/Synchronizer.py

```diff
@@ -214,6 +190,49 @@ def _syncNodes( self ):
 
   ## Private methods ###########################################################
 
+  def __removeNonExistingResources( self ):
+    '''
+      Remove resources that no longer exist in the CS.
+    '''
+
+    url = getServiceURL( "ResourceStatus/ResourceManagement" )
+
+    if not url:
+      gLogger.verbose( 'ResourceManagement is not installed, skipping removal of non existing resources...' )
+      return S_OK()
```

Comment:
wait, I'm not touching `ResourceStatusDB` at all here as far as I know, I'm only removing entries from the `DowntimeCache` table which is part of `ResourceManagementDB`, right?